### PR TITLE
Bump compare-urls to 3.0.0 (security flaw)

### PR DESCRIPTION
**Pull request by BigZ**
# Why

security flaw reported in npm audit

# How
simple bump

# Test Plan
up to you

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.io and README.md).
- [x] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [x] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by bbarthec — CHANGES_REQUESTED** [review]
@BigZ, I need you to address CHANGELOG entry request mentioned by the CI.

File: packages/expo-web-browser/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "expo-web-browser",
-  "version": "9.2.0",
+  "version": "9.2.1",
```
Comment (Simek) [inline packages/expo-web-browser/package.json:3]: Please revert this line, update the PR branch and address the Bartek request, so we can merge this bump in. 😉

**Comment by BigZ** [comment]
hello @Simek  @bbarthec thanks a lot for considering my PR
i did the request change, i hope everything is fine
all the best to you ! greetings !

**Comment by Simek** [comment]
Hey @BigZ, sorry for letting this slip trough, and we did not follow up on that contribution.

Since `expo-web-browser` no longer uses the problematic dependency I think we can close this PR, see:
* https://github.com/expo/expo/blob/337828590b3c7f98339f6dd1ae16d12a2b7ea12c/packages/expo-web-browser/package.json#L35-L43

Thanks anyway!

**Pull request closed without merging**